src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,91 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { TotoActions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const [isLoader, setIsLoader] = useState<boolean>(false);
```

Comment:
To avoid the jumpings that I mentioned before, you can use `true` for the initial value here and set it to false only if the loading ends
```suggestion
  const [isLoader, setIsLoader] = useState<boolean>(true);
```